lib/db.js

```diff
@@ -856,6 +856,49 @@ module.exports = function (
     return this.pool.get('/emailBounces/' + Buffer(email, 'utf8').toString('hex'))
   }
 
+  DB.prototype.accountEmails = function (uid) {
+    log.trace({
+      op: 'DB.accountEmails',
+      uid: uid
+    })
+
+    return this.pool.get('/account/' + uid.toString('hex') + '/emails')
+  }
+
+  DB.prototype.createEmail = function (uid, emailData) {
+    log.trace({
+      op: 'DB.createEmail',
+      uid: emailData.uid
+    })
+
+    return this.pool.post('/account/' + uid.toString('hex') + '/emails', emailData)
+      .catch(
+        function (err) {
+          if (isEmailAlreadyExistsError(err)) {
+            throw error.emailExists()
+          }
+          throw err
+        }
+      )
+  }
+
+  DB.prototype.deleteEmail = function (uid, email) {
+    log.trace({
+      op: 'DB.deleteEmail',
+      uid: uid
+    })
+
+    return this.pool.del('/account/' + uid.toString('hex') + '/emails/' + email)
```

Comment:
Hm should we make `'/emails/' + email` pass the `email` as data instead of part of the url?